triton-vm/src/lib.rs

```diff
@@ -113,3 +113,66 @@ pub fn prove(
 pub fn verify(parameters: &StarkParameters, claim: &Claim, proof: &Proof) -> bool {
     Stark::verify(parameters, claim, proof, &mut None).unwrap_or(false)
 }
+
+#[cfg(test)]
+mod public_interface_tests {
+    use triton_opcodes::program;
+    use twenty_first::shared_math::bfield_codec::BFieldCodec;
+
+    use crate::stark::StarkHasher;
+
+    use super::*;
+
+    #[test]
+    pub fn lockscript_test() {
+        // Program proves the knowledge of a hash preimage
+        let source_code = "
+        divine divine divine divine divine
+        hash pop pop pop pop pop
+        push 09456474867485907852 push 12765666850723567758 push 08551752384389703074 push 03612858832443241113 push 12064501419749299924
```

Comment:
```suggestion
        push 09456474867485907852
        push 12765666850723567758
        push 08551752384389703074
        push 03612858832443241113
        push 12064501419749299924
```
Breaks the line length convention otherwise.